tests/test_analysis_cli.py

```diff
@@ -365,6 +365,47 @@ def test_analysis_get_drafts(cli_run):
     }]
 
 
+@responses.activate
```

Comment:
Lets remove these tests, since they don't really test the new feature, but just that the client will give a response, which is already tested